ruby/vampires.rb

```diff
@@ -1,3 +1,5 @@
+#random comment so I can make a pull request
+
 puts "How many employees will be processed today?"
 new_employees = gets.chomp.to_i
 

```

Comment:
Thanks!  I'll definitely look at the Time class, I figured there was something along those lines but was caught up in getting the basic design down.
In terms of the ordering logic, normally I'd look at the most specific item first (in this case checking the name), and if/elsif from there.  Reading over the instructions I thought it asked for the conditions to be kept in the same order (which disallows this)?  Maybe I misread/misunderstood the initial prompt.